tests/data/output/annotations/basic/output-json-schema.json

```diff
@@ -60,16 +60,14 @@
             "readOnly": true
         },
         "nonreadonly": {
-            "type": "string",
-            "readOnly": false
+            "type": "string"
```

Comment:
this seems to be changing the logic of pass-through - is this intentional ?

Reply:
Yes, if it's a `false` or not set, we shouldn't emit it. It defaults to `false` in the JSON Schema. IOW, emit as smallest JSON Schema as possible. Is it ok?

Reply:
see other comment